### DISCUSSION TITLE
This library is an opensource joke ?

### DISCUSSION THREAD

**Discussion by andreivips** [pmndrs/use-cannon#165] [General] [answered]
Its not experimental, it has major bugs left to avoid serious usage of this lib, yes ? Why then...

Meanwhile r3f is just great, and this one is show off or something.
https://github.com/pmndrs/use-cannon/issues/115
**next, explore...**

**Comment by stockhuman** [comment, marked as answer]
We welcome your pull request, then.

**Comment by andreivips** [comment]
Its a long root. Somehow in core...

**Reply by stockhuman** [reply]
For reference, I've converted this to a discussion, because it is one. If you would like to get bugs fixed faster: open a pull request with your fix. Complaining via issues is not open-source, and you must realize that everyone onboard, with few exceptions, is doing this for free, on their own time.

Please note that r3f is effectively a flagship, with a very large community, playing an integral role in many projects. Poimandres concentrates their effort there with good reason. use-cannon on the other hand is a niche adapter to a fork of a physics library. I was around at the start of it, tended to my life whilst other maintainers kept up the pace, and have only recently returned to slowly carve away at the backlog. I can only dedicate so much time to this library, and so is the case with everyone else that contributes.

If you would like to join that group of people and count yourself amongst the contributors, you are very welcome to - that's the spirit of open-source. Else, please - insults don't make free programming happen faster.

**Reply by andreivips** [reply]
Sorry, maybe i am running to fast some own stuff while using this experiment.
